examples/step_count/run.py

```diff
@@ -0,0 +1,198 @@
+import random
+from collections.abc import Iterator
+from typing import Self, override
+
+import numpy as np
+import polars as pl
+from numpy.typing import NDArray
+
+from flowcean.cli.logging import initialize_logging
+from flowcean.environments.hybrid_system import (
+    DifferentialMode,
+    HybridSystem,
+    State,
+)
+from flowcean.environments.train_test_split import TrainTestSplit
+from flowcean.learners.regression_tree import RegressionTree
+from flowcean.metrics.regression import MeanAbsoluteError, MeanSquaredError
+from flowcean.strategies.offline import evaluate_offline, learn_offline
+from flowcean.transforms.sliding_window import SlidingWindow
+
+class Acceleration(State):
```

Comment:
Is the acceleration the state or the external input. I would rather expect the acceleration to be the input and the state being the velocity and / or the number of steps counted.

Reply:
Yes, changes are made accordingly. Acceleration is the state. acceleration is the variable that governs the continuous dynamics that is randomly generated.